diff -urN gabber-0.8.4.org/configure.in gabber-0.8.4/configure.in
--- gabber-0.8.4.org/configure.in	Wed Aug 29 23:55:24 2001
+++ gabber-0.8.4/configure.in	Wed Aug 29 23:55:45 2001
@@ -60,7 +60,7 @@
 dnl * Here are the LANGuages
 dnl ******************************
 ALL_LINGUAS="az da de es fi fr ko nl no ru sv tr uk"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 EVO_CHECK_LIB(libglade, 0.11, libglade)
 EVO_CHECK_LIB(GAL, 0.3, gal)
