diff -Nuard gabber-0.8.7.orig/doc/C/gabber-C.omf gabber-0.8.7/doc/C/gabber-C.omf
--- gabber-0.8.7.orig/doc/C/gabber-C.omf	Mon Feb 26 21:32:56 2001
+++ gabber-0.8.7/doc/C/gabber-C.omf	Thu Oct  3 18:45:05 2002
@@ -1,14 +1,22 @@
 <?xml version="1.0" standalone="no"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       The Gabber Manual
     </title>
-    <subject>
-      <category>GNOME|Network</category>
-    </subject>
+    <date>
+	2002-10-03
+    </date>
+    <version identifier="0.8.7" date="2002-10-03"/>
+    <subject category="GNOME|Network"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="gabber.sgml"/>
     <language code="C"/>
+    <relation seriesid="3cbb94e8-d6ef-11d6-8291-d6b8b1575a19"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
