diff -durN gabber-0.8.6.orig/doc/C/Makefile.am gabber-0.8.6/doc/C/Makefile.am
--- gabber-0.8.6.orig/doc/C/Makefile.am	Wed Jan  2 21:45:03 2002
+++ gabber-0.8.6/doc/C/Makefile.am	Sat Jan 12 13:13:50 2002
@@ -51,11 +51,11 @@
 	$(mkinstalldirs) $(gabber_helpdir)/stylesheet-images
 	-for file in $(SGML_FILES) $(srcdir)/gabber/*.html $(srcdir)/gabber/*.css $(srcdir)/*.png; do \
 	basefile=`echo $$file | sed -e 's,^.*/,,'`; \
-	$(INSTALL_DATA) $(srcdir)/$$file $(gabber_helpdir)/$$basefile; \
+	$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(gabber_helpdir)/$$basefile; \
 	done
 	-for file in $(srcdir)/gabber/stylesheet-images/*.*; do \
 	basefile=`echo $$file | sed -e 's,^.*/,,'`; \
-	$(INSTALL_DATA) $(srcdir)/$$file $(gabber_helpdir)/stylesheet-images/$$basefile; \
+	$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(gabber_helpdir)/stylesheet-images/$$basefile; \
 	done
 
 gabber.ps: $(SGML_FILES)
diff -durN gabber-0.8.6.orig/pixmaps/Makefile.am gabber-0.8.6/pixmaps/Makefile.am
--- gabber-0.8.6.orig/pixmaps/Makefile.am	Sun Nov  4 00:10:40 2001
+++ gabber-0.8.6/pixmaps/Makefile.am	Sat Jan 12 13:14:46 2002
@@ -72,7 +72,7 @@
 glade_DATA = $(glade_pixmaps)
 
 install-exec-hook: 
-	(cd $(datadir)/gabber;	\
+	(cd $(DESTDIR)$(datadir)/gabber;	\
 	ln -sf ../pixmaps/gnome-info.png gnome-info.png;		\
 	ln -sf ../pixmaps/gnome-lockscreen.png gnome-locksreen.png;	\
 	ln -sf ../pixmaps/gnome-globe.png gnome-globe.png;		\
